python/lsst/analysis/tools/atools/coaddInputCount.py

```diff
@@ -0,0 +1,83 @@
+# This file is part of analysis_tools.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+from __future__ import annotations
+
+__all__ = ("CoaddInputCount",)
+
+from ..actions.plot.calculateRange import MinMax
+from ..actions.plot.skyPlot import SkyPlot
+from ..actions.scalar.scalarActions import MeanAction, MedianAction, SigmaMadAction
+from ..actions.vector import CoaddPlotFlagSelector, LoadVector, SnSelector
+from ..interfaces import AnalysisTool
+
+
+class CoaddInputCount(AnalysisTool):
```

Comment:
Please add a class docstring that explains what this is counting and why somebody might care. Docs: https://developer.lsst.io/python/numpydoc.html#py-docstring-class-structure